src/app/components/CallToActionLinkComponent/Text/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
+
+import { PropsWithChildren } from 'react';
+import { jsx } from '@emotion/react';
+import TEXT from '#app/components/Text';
+import { FontVariant, GelFontSize } from '../../../models/types/theming';
+import styles from './index.styles';
+
+type TextProps = {
+  as?: string;
+  fontVariant?: FontVariant;
+  size?: GelFontSize;
+  className?: string;
+  overrideInteractionStyles?: boolean;
+};
+
+const Text = ({
+  as,
+  size,
+  fontVariant,
```

Comment:
In the original component -  https://github.com/bbc/simorgh/blob/latest/src/app/components/CallToActionLink/index.tsx#L27, the default was `sansBold`

```suggestion
  fontVariant = 'sansBold',
```

Reply:
Makes sense - it is the most widely used implementation